react/src/components/Chat/Conversation.tsx

```diff
@@ -0,0 +1,126 @@
+import Flex from '../Flex';
+import ChatCard from './ChatCard';
+import { ChatProviderType, ChatType, ConversationType } from './ChatModel';
+import { ConversationQuery } from './__generated__/ConversationQuery.graphql';
+import { useDynamicList } from 'ahooks';
+import { Card, Skeleton } from 'antd';
+import graphql from 'babel-plugin-relay/macro';
+import { map } from 'lodash';
+import { Suspense, useId } from 'react';
+import { useLazyLoadQuery } from 'react-relay';
+
+export type ConversationProps = {
+  conversation: ConversationType;
+  provider: ChatProviderType;
+};
+
+const ChatViewStyle = {
+  overflow: 'auto',
+  height: 'calc(100vh - 240px)',
+};
+
+const ChatSkeletonStyle = {
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column' as const,
+};
+
+const ChatCardStyle = { flex: 1, overflow: 'hidden' };
+
+function useSelectedEndpoint(endpointId?: string) {
+  const { endpoint, endpoint_list } = useLazyLoadQuery<ConversationQuery>(
+    graphql`
+      query ConversationQuery(
+        $endpointId: UUID!
+        $isEmptyEndpointId: Boolean!
+      ) {
+        endpoint(endpoint_id: $endpointId)
+          @skipOnClient(if: $isEmptyEndpointId)
+          @catch {
+          endpoint_id
+          ...ChatCard_endpoint
+        }
+        endpoint_list(limit: 1, offset: 0) {
+          items {
+            endpoint_id
+            ...ChatCard_endpoint
+          }
+        }
+      }
+    `,
+    {
+      endpointId: endpointId || '',
+      isEmptyEndpointId: !endpointId,
+    },
+  );
+
+  return (endpoint.ok ? endpoint.value : endpoint_list?.items?.[0]) ?? null;
+}
+
+function createNewChat(
+  id: string,
+  conversationId: string,
+  provider: ChatProviderType,
+) {
+  return {
+    id,
+    conversationId,
+    label: 'Chat',
+    sync: false,
+    provider,
+  };
+}
+
+const ConversationStyle = {
+  overflow: 'hidden',
+};
```

Comment:
If you are going to separate inline styles, how about using the ant style approach? You can refer to https://github.com/ant-design/antd-style?tab=readme-ov-file#create-styles.
WebUI already user `antd-style`